src/components/Post/PostHeading.tsx

```
@@ -262,18 +263,51 @@ const PostHeading: FC<IPostHeadingProps> = (props) => {
+										<BeneficiaryAmoutTooltip
+											key={beneficiary?.address}
+											assetId={beneficiary?.genralIndex || null}
+											requestedAmt={beneficiary?.amount.toString()}
+											className={classNames(className, 'flex')}
+											postId={onchainId ? Number(onchainId) : (onchainId as any)}
+											proposalCreatedAt={created_at as any}
+											timeline={timeline || []}
+											usedInPostPage
+										/>
+									);
+								})}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)